docs/docusaurus/docs/cloud/schedules/manage_schedules.md

```diff
@@ -0,0 +1,75 @@
+---
+sidebar_label: 'Manage schedules'
+title: 'Manage schedules'
+description: Create and manage schedules for Validations in GX Cloud.
+---
+
+Add schedules to your GX Cloud Expectation Suites to take advantage of automated data quality checks. Schedules are automatically enabled when you add your first Expectation to and Expectation Suite in the Cloud UI.
+
+## Create a schedule for an existing Expectation Suite
+
+1. In GX Cloud, click **Data Assets**.
+
+2. In the **Data Assets** list, click the Data Asset name.
+
+3. Click the **Expectations** tab and then select the Expectation Suite you'd like to apply a schedule to.
+
+4. Ensure you have an Expectation in the Expectation Suite. Unpause the schedule for the Expectation Suite using the toggle in the Scheduling component.
+
+5. To edit the default schedule, click the edit icon in the Scheduling component.
+
+    - **Frequency:** Select a frequency between 1 hour and 24 hours.
+
+    - **Start time:** Select a local start time for your schedule. Your selected frequency overrides start time, in the case where start time is later than the next occurence of selected frequency.
```

Comment:
This feels a little ambiguous -- at first I thought "does that mean that if I set a frequency of 1 hour and a start time three hours from now, it's going to run in 1 hour from now instead?"  That didn't seem sensible, though.  Do you mean to convey that the start time determines the first time the scheduler runs, and all subsequent run times will be determined by the selected frequency and the previous run time? (i.e. the schedule won't "restart" when the start time rolls around again on a future date.)

Reply:
I struggled to phrase this, because I agree that the start time _should_ determine the first time the scheduler runs, and all subsequent run times are determined by selected frequency. For MVP however, the frequency determines when the first run is, based off the start time selected. The scenario you describe is correct. Another example is "if it's 12pm and I set a frequency of 6 hours and a start time of 9pm, the next run is at 3pm today.